device/common/include/traccc/clusterization/device/aggregate_cluster.hpp

```diff
@@ -38,7 +44,9 @@ inline void aggregate_cluster(
     const silicon_detector_description::const_device& det_descr,
     const vecmem::device_vector<details::index_t>& f, unsigned int start,
     unsigned int end, unsigned short cid, measurement& out,
-    vecmem::data::vector_view<unsigned int> cell_links, unsigned int link);
+    vecmem::data::vector_view<unsigned int> cell_links, unsigned int link,
+    vecmem::device_vector<unsigned int> disjoint_set,
```

Comment:
Crud... I just now see that it should rather be

```suggestion
    const vecmem::device_vector<unsigned int>& disjoint_set,
```

:frowning: